### ISSUE TITLE
EbayFakeMenuButton: wrong usage of aria-haspopup attribute

### ISSUE THREAD

**Issue by HenriqueLimas** [eBay/evo-web#695]
### I verified there's no existing issue for this bug.

- [x] There are no existing issues

### Current behavior

[Fake Menu Button](https://ebay.gitbook.io/mindpatterns/navigation/fake-menu-button) are not considered as a opening it in a menu or dialog, so they shouldn't have `aria-haspopup` as it is announcing that the user can navigate with arrow keys, but it cannot.

### Expected behavior

`aria-haspopup` shouldn't be used in Fake menu button

### Steps to reproduce

```bash
1.
2.
3.
...
```

**Comment by ArtBlue** [comment]
This issue has a merged PR so the work has already been completed.
